index.html

```diff
@@ -4,9 +4,146 @@
+                                    <li class="navbar-bottom-item"><a href="#" class="navbar-bottom-item-link">Join</a></li>
+                                    <li class="navbar-bottom-item"><a href="#" class="navbar-bottom-item-link last">IES Campaign</a></li>
+                                </ul>
+                            </nav>
+                        </div>
+            </div>
+    </header>
+    <main class="main">
+        <div class="headline-container">
+            <section class="headline container">
+                <div class="headline-title container">
+                    <h2>“Hello! Bee Hive”</h2>
+                    <h1>Industrial Engineering Symposium 2022</h1>
+                    
+                </div>
+                <p>Next April 21, 22, 23 and 24, at the Hotel Lina Congress Center, the 2022 edition of the leading event of the Dominican automotive industry will be held.</p>
+                <h3 >2022.04.21(THU) ~ 24(SUN)</h3>
+                <h4>&commat; Hotel Lina Congress Center</h4>
+            </section>
+        </div>
+        <section class="program-section">
+            <h2>Main Program</h2>
+            <hr/>
+            <div class="cards-program container">
+                <article class="program-card">
+                    <div class="program-card-headline">
+                        <img src="images/Lecture.png" alt="lecture">
+                        <h3>Lecture</h3>
+                    </div>
+                    <div class="program-card-description">
+                        <p>Listen to the speckers from various countries about the message of Automotive Manufacturing.</p>
+                    </div>
+                    
+                </article>
+                <article class="program-card">
+                    <div class="program-card-headline">
+                    <img src="images/Exhibition.png" alt="exhibition">
+                    <h3>Exhibition</h3>
+                    </div>
+                    <div class="program-card-description">
+                        <p>Appreciate illustrations from experts and students in the field of Automative Manufacturing.</p>
+                    </div>
+                    
+                </article>
+                <article class="program-card">
+                    <div class="program-card-headline">
+                        <img src="images/Forum.png" alt="forum">
+                        <h3>Forum</h3>
+                    </div>
+                    <div class="program-card-description">
+                        <p>Have time to share your thoughts and opinions with experts for each topic.</p>
+                    </div>
+
+                    
+                </article>
+                <article class="program-card">
+                    <div class="program-card-headline">
+                        <img src="Images/Workshop.png" alt="workshop">
```

Comment:
- [ ] Please take care of your images, right now they are not being shown, probably because something changed on your path to those images. Take a look at all of them, because no image is being shown.

Reply:
Hello @Lordkaito, thanks for your response. Kindly I can see the image locally, I don't know what happened.